golang/cosmos/Makefile

```diff
@@ -116,34 +116,37 @@ SDK_UPGRADE_TYPES = third_party/proto/cosmos/upgrade/v1beta1
 
 proto-update-deps:
 	mkdir -p $(COSMOS_PROTO_TYPES)
-	curl -sSL $(COSMOS_PROTO_PROTO_URL)/cosmos.proto > $(COSMOS_PROTO_TYPES)/cosmos.proto
+	url="$(COSMOS_PROTO_PROTO_URL)"; \
+	curl -sSL $$url/cosmos.proto > $(COSMOS_PROTO_TYPES)/cosmos.proto
 
 	mkdir -p $(GOGO_PROTO_TYPES)
-	curl -sSL $(GOGO_PROTO_URL)/gogoproto/gogo.proto > $(GOGO_PROTO_TYPES)/gogo.proto
+	url="$(GOGO_PROTO_URL)"; \
+	curl -sSL $$url/gogoproto/gogo.proto > $(GOGO_PROTO_TYPES)/gogo.proto
 
 	# Downloading from a not-a-go-module is problematic. These files are artifacts for now.
 	# mkdir -p $(GOOGLE_API_TYPES)
-	# curl -sSL $(GOOGLE_API_URL)/annotations.proto > $(GOOGLE_API_TYPES)/annotations.proto
-	# curl -sSL $(GOOGLE_API_URL)/http.proto > $(GOOGLE_API_TYPES)/http.proto
-	# curl -sSL $(GOOGLE_API_URL)/httpbody.proto > $(GOOGLE_API_TYPES)/httpbody.proto
+	# url="$(GOOGLE_API_URL)"; \
+	# curl -sSL $$url/annotations.proto > $(GOOGLE_API_TYPES)/annotations.proto && \
+	# curl -sSL $$url/http.proto > $(GOOGLE_API_TYPES)/http.proto && \
+	# curl -sSL $$url/httpbody.proto > $(GOOGLE_API_TYPES)/httpbody.proto &&
 
 	mkdir -p $(GOOGLE_PROTO_TYPES)
-	curl -sSL $(GOGO_PROTO_URL)/protobuf/google/protobuf/any.proto > $(GOOGLE_PROTO_TYPES)/any.proto
-
-	mkdir -p $(IBC_CHANNEL_TYPES)
-	curl -sSL $(IBC_PROTO_URL)/channel/v1/channel.proto > $(IBC_CHANNEL_TYPES)/channel.proto
-
-	mkdir -p $(IBC_CLIENT_TYPES)
-	curl -sSL $(IBC_PROTO_URL)/client/v1/client.proto > $(IBC_CLIENT_TYPES)/client.proto
-
-	mkdir -p $(SDK_BASE_TYPES)
-	curl -sSL $(COSMOS_SDK_PROTO_URL)/base/v1beta1/coin.proto > $(SDK_BASE_TYPES)/coin.proto
-
-	mkdir -p $(SDK_QUERY_TYPES)
-	curl -sSL $(COSMOS_SDK_PROTO_URL)/base/query/v1beta1/pagination.proto > $(SDK_QUERY_TYPES)/pagination.proto
-
-	mkdir -p $(SDK_UPGRADE_TYPES)
-	curl -sSL $(COSMOS_SDK_PROTO_URL)/upgrade/v1beta1/upgrade.proto > $(SDK_UPGRADE_TYPES)/upgrade.proto
+	url="$(GOGO_PROTO_URL)"; \
+	curl -sSL $$url/protobuf/google/protobuf/any.proto > $(GOOGLE_PROTO_TYPES)/any.proto
```

Comment:
Profoundly uncomfortable with double dollar signs in principle (and not practice.)